apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -247,6 +252,8 @@ describe("Client API", () => {
   });
 
   it("creates a new client, accesses institution cache endpoint, and deletes the client", () => {
+    cy.wait(10000);
```

Comment:
Without this wait, this series of request would fail the majority of the time, with a rate limit error.